master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,167 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``0.9.4-111-g93f4b41`` ( ``2017-02-16`` )
```

Comment:
towncrier will modify this file when doing "make docs". Please do not include it in your commits